### DISCUSSION TITLE
Is Envault now free?

### DISCUSSION THREAD

**Discussion by rconte84** [envault/envault#21] [Q&A] [answered]
Hi all! I'm a bit confuse... Envault is now free? I have bought it 4/5 mounth ago...

**Comment by danharrin** [comment, marked as answer]
Hi @rconte84!

Yes you're correct, as per the announcement email that you should have received as an initial backer, we have released v2 as an open source product.

You can continue to use it as you did previously, and take advantage of premium support (contactable via hello@envault.dev) as you were originally promised.

Many thanks for backing the project in it's early stages!

**Comment by rconte84** [comment]
All right! Thanks for all your work.

Il lun 4 gen 2021, 12:13 Dan Harrin <notifications@github.com> ha scritto:
